examples/pylint/R0916_too_many_boolean_expressions.py

```diff
@@ -0,0 +1,6 @@
+def numberChoice(x: int, y: int, z: int) -> None:
```

Comment:
Use `number_choice` here (and in general convert "CamelCase" to "snake_case" for function/variable names.